trillian/ctfe/handlers.go

```diff
@@ -734,11 +747,15 @@ func getEntries(ctx context.Context, li *logInfo, w http.ResponseWriter, r *http
 			return li.toHTTPStatus(err), fmt.Errorf("backend GetLeavesByIndex request failed: %s", err)
 		}
 
-		if rsp.SignedLogRoot != nil && rsp.SignedLogRoot.TreeSize <= start {
+		var currentRoot types.LogRootV1
```

Comment:
No change needed here, but I notice that `TrillianLogRPCServer.GetLeavesByIndex()` doesn't do the `root.UnmarshalBinary` check that the other entrypoints do &ndash; could you make it consistent in the Trillian PR?

Reply:
I've got a trillian PR in progress to remove the fields completely. However it tries to build this code so I've done this side first.

Reply:
Ah that breaks `sth_test` because I shared test cases for log / mirror and there's no `testing.T` when they're declared.